.github/workflows/cron.yml

```diff
@@ -0,0 +1,154 @@
+name: Cron
+
+on:
+  schedule:
+    - cron: '0/10 * * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  install:
+    name: 'Install'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check cache for "node_modules"
+        id: cache
+        uses: martijnhols/actions-cache/check@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+
+      - name: Save "node_modules" to cache
+        if: steps.cache.outputs.cache-hit != 'true'
+        uses: martijnhols/actions-cache/save@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ steps.cache.outputs.primary-key }}
+
+  fetchWhitelist:
+    name: 'Fetch whitelist'
+    runs-on: ubuntu-latest
+    environment: CI
+    needs: install
+    permissions:
+      id-token: write # need this for OIDC
+      contents: read # This is required for actions/checkout@v2
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-region: 'us-east-2'
+          aws-access-key-id: ${{ secrets.AWS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+      - name: 'Fetch'
+        run: aws s3 cp s3://${{ secrets.AWS_BUCKET }}/ArbTokenLists/arbed_arb_whitelist_era.json ./src/ArbTokenLists/arbed_arb_whitelist_era.json
+
+      - uses: actions/cache@v3
+        id: cache-whitelist
+        with:
+          path: ./src/ArbTokenLists/arbed_arb_whitelist_era.json
+          key: ${{ github.sha }}
+
+  generate-token-list:
+    name: 'Generate token list'
```

Comment:
nit: shorten to just `Generate`, it should be cleaner when you're viewing the entire matrix of jobs in GitHub Actions

Reply:
done